apps/3000-home/pages/index.tsx

```diff
@@ -1,19 +1,10 @@
-import React, { Suspense } from 'react';
+import React, { Suspense, lazy } from 'react';
 import Head from 'next/head';
-import dynamic from 'next/dynamic';
 
-const CheckoutTitle = dynamic(() => import('checkout/CheckoutTitle'), {
-  ssr: true,
-});
-const ButtonOldAnt = dynamic(() => import('checkout/ButtonOldAnt'), {
-  ssr: true,
-});
-const WebpackSvgRemote = dynamic(() => import('shop/WebpackSvg'), {
-  ssr: true,
-});
-const WebpackPngRemote = dynamic(() => import('shop/WebpackPng'), {
-  ssr: true,
-});
+const CheckoutTitle = lazy(() => import('checkout/CheckoutTitle'));
```

Comment:
use of lazy here seems to stop hydration issues, dynamic attempts to preload them then start rendering - which can cause a state change half way through

Reply:
I can fix this, i think - since its related to chunk flushing. 
For now just use dynamic